salt/cloud/clouds/vmware.py

```diff
@@ -471,20 +473,29 @@ def _add_new_scsi_controller_helper(scsi_controller_label, properties, bus_numbe
     return scsi_spec
 
 
-def _add_new_ide_controller_helper(ide_controller_label, properties, bus_number):
+def _add_new_ide_controller_helper(ide_controller_label, controller_key, bus_number):
     '''
     Helper function for adding new IDE controllers
 
     .. versionadded:: 2016.3.0
+
```

Comment:
We need to make the documentation for internal functions more consistent and possibly auto generated based on the function signature. No need to add docs for this function for now since other functions are missing docs.